front/app/modules/commercial/id_vienna_saml/components/ViennaSamlButton.tsx

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
 import { AUTH_PATH } from 'containers/App/constants';
+import React from 'react';
 
 // i18n
-import messages from './messages';
 import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
 
-import { StyledAuthProviderButton } from 'components/SignUpIn/AuthProviders';
+// components
 import { TOnContinueFunction } from 'components/SignUpIn/AuthProviderButton';
+import { StyledAuthProviderButton } from 'components/SignUpIn/AuthProviders';
+import ViennaIcon from './ViennaIcon';
 
 // typings
 import { TSignUpInFlow } from 'components/SignUpIn';
 
+// styling
+import styled from 'styled-components';
+import { fontSizes } from 'utils/styleUtils';
+
+const Container = styled.div`
+  display: flex;
+  gap: 10px;
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  gap: 2px;
+  flex-direction: column;
+`;
+
+const SignUpSubHeader = styled.span`
```

Comment:
You can use our Text component for this (`as="span"`). We are trying to move away from using so many styled components all over the place.